frontend/integration-tests/views/crud.view.ts

```diff
@@ -167,3 +172,35 @@ export const emptyState = $('.cos-status-box').$('.text-center');
 
 export const errorMessage = $('.alert-danger');
 export const successMessage = $('.alert-success');
+
+export const createNamespacedResourceWithDefaultYaml = async function(resourcetype:string) {
+  await browser.get(`${appHost}/k8s/ns/${testName}/${resourcetype}`);
+  await isLoaded();
+  const createDropdownIsPresent = await createItemButton.isPresent();
+  if (createDropdownIsPresent) {
+    await createItemButton.click();
+    await createYAMLLink.click();
+  } else {
+    await browser.wait(until.presenceOf(createYAMLButton));
+    await createYAMLButton.click();
+  }
+  browser.wait(until.and(untilNoLoadersPresent, until.presenceOf(element(by.cssContainingText('h1', 'Create')))));
+  await browser.wait(until.presenceOf(yamlView.saveButton));
+  await yamlView.saveButton.click();
+};
+
+export const addNewStorageToWorkload = async function(pvcName:string, pvcSize:string, mountPath:string) {
+  await browser.wait(until.presenceOf(actionsButton));
+  await actionsButton.click();
+  await browser.wait(until.presenceOf(addStorageItem));
+  await addStorageItem.click();
```

Comment:
There are helpers for using actions already in crud view. Use the helper.

Reply:
Updated.